invenio/modules/pages/static/css/pages/list.less

```diff
@@ -0,0 +1,70 @@
+.guide-listing {
+  width: 440px;
+  margin-right: 50px;
+  margin-bottom: 30px;
+}
+
+.guide-cover {
+  position: relative;
+  z-index: 2;
+  height: 225px;
+  color: #fff;
+  text-align: center;
+  background-color: #aaa;
+  background: url(http://www.noisetexturegenerator.com/images/bg.png) center top repeat;
```

Comment:
we should not include resources from external urls. can you create a static file?